mantidproject/mantid
# Dev-docs: Git and coding standards-related pages

Pages associated with Git, GitHub and coding standards were moved to the new `dev-docs` target

**To test:**

Build the docs and have a quick read through them to make sure everything looks correct and sounds sensible

Refs #22082 

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
